docker/repo-server-controller/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.1.0-1760
+
+LABEL maintainer="Tom Manville<tom@kasten.io>"
+
+RUN microdnf -y update openssl-libs
```

Comment:
Why do we need to run this command?

Reply:
Even I am unsure of that, I followed the dockerfile of kanister controller

Reply:
This PR has details about why it was updated in kanister controller, since we are using same base image for repo server controller, I kept it as it is:
https://github.com/kanisterio/kanister/pull/949